packages/email/src/components/Header.mjml

```diff
@@ -0,0 +1,19 @@
+import mjml2html from 'mjml';
+
+const htmlOutput = mjml2html(`
+<mjml>
+  <mj-body>
+    <mj-wrapper full-width="full-width" padding="0 35px 0 35px">
+      <mj-section background-color="#e8edf5" padding="25px">
+        <mj-column>
+          <mj-text font-weight="700" color="black" css-class="inline" align="center">add
+            <img width="25px" src="https://github.com/CatalysmsServerManager/frontend/blob/master/packages/email/src/images/csmm-icon-original.svg?raw=true" />
```

Comment:
Interesting! This makes me think we might need a proper CDN in the future instead of hotlinking to Github :smile:

Reply:
If we ever make this repo private for whatever reason, this'll break so gotta be mindful of that